diff --git a/docs/radar.js b/docs/radar.js
index 65f40a80d327c6860bdc3dc7f5a90b5274159715..469aa4313706fc49f51888e7c6aaff195449b27c 100644
--- a/docs/radar.js
+++ b/docs/radar.js
@@ -20,9 +20,9 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.

+import * as d3 from "d3"

-function radar_visualization(config) {
-
+export function radar_visualization(config) {
   config.svg_id = config.svg || "radar";
   config.width = config.width || 1450;
   config.height = config.height || 1000;
